src/gssapi_client.cpp

```diff
@@ -51,8 +51,12 @@ zmq::gssapi_client_t::gssapi_client_t (const options_t &options_) :
     service_name = static_cast <char *>(malloc(service_size+1));
     assert(service_name);
     memcpy(service_name, options_.gss_service_principal.c_str(), service_size+1 );
-    service_name_type = convert_nametype (options_.gss_service_principal_nt);
 
+#ifdef ZMQ_BUILD_DRAFT_API
```

Comment:
The definitions are available internally, so you can remove all te \#ifdefs in the internal code - this is the reason we added src/zmq_drafts.h, so that internal code would be easier to write and maintain without lots and lots of ifdefs.
If users manually inspect the code and find out the magic number and use it it's their problem anyway - we guarantee a stable API, not safety from silly hacks